ItemList/ItemList/ViewModel.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace ItemList
+{
+    /// <summary>
+    /// This is a class, that represents a ViewModel layer with all fields from View and logic how to handle requests from UI
```

Comment:
Don't forget to put dot in the end of each sentence. This means you've completed your comment.